README.NL.rst

```diff
@@ -0,0 +1,55 @@
+=====================
+Record Management App
+=====================
+
+:Version: 0.1.0
+:Source: https://github.com/GemeenteUtrecht/record-management-app
+:Keywords: Common Ground, Record Management, Archiveren
+:PythonVersion: 3.8
+
+Zaakgericht record-management in het `Common Ground`_ landschap.
+
+Ontwikkeld door `Maykin Media B.V.`_ voor Gemeente Utrecht en Gemeente Delft.
+
+
+Introductie
+===========
+
+Record management is the practice of destroying data-records that have met their
+end-of-life time. "Zaakgericht werken" prescribes the archiving terms for cases that
+have been brought to an end.
+
+The RMA provides functionality for record managers to schedule destruction of cases
+according to the archiving parameters. It implements a multi-step, multi-role accordance
+process.
+
+Attention is paid to the user experience of the staff using the app, with notifications
+to inform you of pending work-load, full audit logging and traceability of user and
+system actions.
+
+The RMA supports all backends implementing the 1.0.x `API's voor Zaakgericht Werken`_.
```

Comment:
Archiefbeheer draait om het vernietigen van gegevens die aan het einde van hun levensduur zijn gekomen. Zaakgericht werken schrijft de archiveringsvoorwaarden voor, voor zaken die zijn beëindigd.

De Record Management App (RMA) biedt functionaliteit voor archiefbeheer om de vernietiging van zaken in te plannen
volgens de archiveringsvoorwaarden. De RMA volgt een proces dat uit meerdere stappen bestaat, over verschillende rollen.

Er is aandacht besteed aan de gebruikerservaring van de beheerders die de app gebruiken, met notificaties
om te informeren over lopende taken, volledige auditregistratie en traceerbaarheid van gebruiker- en
systeemacties.

De RMA ondersteunt alle backends die de 1.0.x  `API's voor Zaakgericht Werken`_ implementeren.